src/main/protobuf/ucfg.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+option java_package = "org.sonar.ucfg.protobuf";
+option optimize_for = SPEED;
+
+message UCFG {
+    string methodId = 1;
+    Location location = 2;
+    repeated string parameters = 3;
+    repeated string ucfgEntries = 4;
+    repeated BasicBlock basicBlocks = 5;
+}
+
+message BasicBlock {
+    string id = 1;
+    Location location = 2;
+    repeated Instruction instruction = 3;
```

Comment:
@benzonico should we rename `Instruction` to `AssignCall`?

Reply:
I would prefer to keep the right to extend further things if required. I agree that now this is exactly the same thing.